src/main/java/org/opentripplanner/graph_builder/GraphBuilder.java

```diff
@@ -126,10 +127,18 @@ public static GraphBuilder create(
             osmModule.banDiscouragedWalking = config.banDiscouragedWalking;
             osmModule.banDiscouragedBiking = config.banDiscouragedBiking;
             graphBuilder.addModule(osmModule);
-            PruneFloatingIslands pruneFloatingIslands = new PruneFloatingIslands();
-            pruneFloatingIslands.setPruningThresholdIslandWithoutStops(config.pruningThresholdIslandWithoutStops);
-            pruneFloatingIslands.setPruningThresholdIslandWithStops(config.pruningThresholdIslandWithStops);
-            graphBuilder.addModule(pruneFloatingIslands);
+            if (OTPFeature.PruneIslands.isOn()) { // original pruning
```

Comment:
I'm not sure what the guideline is but I find it a little strange to configure the pruning in both `otp-config.json` and `build-config.json`. WDYT?

Reply:
Would it make sense to put all the config in the build one?

Reply:
I do not know. Feature switches seem to be collected nowadays into otp config.

Reply:
All OTPFeatures need to be configured for the build and the router, so the `otp-config.json` should be the same for building the graph and serving it. I agree that one file would be enough, and then you can chose to include everything or just build/router config.

Reply:
Need for 2 configs is now removed. Pruning is a standard feature.